cypher-shell/src/test/java/org/neo4j/shell/cli/InteractiveShellRunnerTest.java

```diff
@@ -248,64 +262,67 @@ public void emptyLineIsIgnored() throws Exception {
     public void testPrompt() throws Exception {
         // given
         InputStream inputStream = new ByteArrayInputStream("".getBytes());
-        InteractiveShellRunner runner = new InteractiveShellRunner(cmdExecuter, txHandler, logger, statementParser, inputStream, historyFile, userMessagesHandler);
+        InteractiveShellRunner runner = new InteractiveShellRunner(cmdExecuter, txHandler, databaseManager, logger, statementParser, inputStream,
+                historyFile, userMessagesHandler, connectionConfig);
 
         // when
         when(txHandler.isTransactionOpen()).thenReturn(false);
         AnsiFormattedText prompt = runner.getPrompt();
 
         // then
-        assertEquals("neo4j> ", prompt.plainString());
+        assertEquals( format("myusername@mydb%n> "), prompt.plainString());
```

Comment:
Perhaps the newline formatting should be configurable, not everyone will want that. I, for one, like single-line prompts.

Reply:
I agree this looks like it should look weird?